package.json

```
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "publish": "lerna publish from-package",
-    "setup:project": "npx pnpm i && node scripts/setup-env-files.js",
+    "setup:project": "npx pnpm i && node scripts/setup-env-files.js && npm run build",
```

```suggestion
    "setup:project": "npx pnpm i && node scripts/setup-env-files.js && npx pnpm build",
```

Maybe it's better to use npx with pnpm in case a new person onboarding the project does not have a global `pnpm` installed.